.github/workflows/chromatic.yml

```diff
@@ -24,3 +24,4 @@ jobs:
         with:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           token: ${{ secrets.GITHUB_TOKEN }}
+          autoAcceptChanges: main
```

Comment:
Chromatic docs seems to implies you can pass a branch name here instead of `true` or `false` but it's not exactly clear, so this might not work.

https://www.chromatic.com/docs/github-actions